### ISSUE TITLE
Bug 8780801: A11y_OneNoe Clipper_Verify accessibility for Sign In to the Clipper with Sign in using a test account_High Contrast_Desert theme: In high contrast desert theme, keyboard focus is not visible on the sign in options.

### ISSUE THREAD

**Issue by aanchalbhansali** [OneNoteDev/WebClipper#608]
Environment Details: 
Application Name: OneNote Clipper-Web
OS: Windows 11 OS build 26058.1000
Edge Dev Version: Version 123.0.2400.1 (Official build) dev (64-bit)
AT's: High Contrast->Desert theme.
Pre-Requisites:
Open the URL: [OneNote Web Clipper Installation](https://www.onenote.com/clipper)
Invoke the 'Get OneNote web Clipper for Microsoft Edge' button.
Edge Add-ons page will appear. Invoke the Get button.
OneNote Clipper will be added in edge extension section. 
Expand the Extension button and turn on the OneNote Clipper Show in Toolbar button.
OneNote Clipper will appear beside of the search field.
Now invoke the OneNote clipper button.
Turn on High contrast desert theme.
Settings->Accessibility->Contrast themes->Desert theme.
Repro Steps:
Open the [Ankylosaurus - Wikipedia](https://en.wikipedia.org/wiki/Ankylosaurus)
Invoke the OneNote Clipper button.
OneNote clipper dialog box will appear with sign in options.
Navigate to the sigh in options and observe the issue.
Actual Result:
In high contrast desert theme, keyboard focus is not visible on the sign in options.
Note: Issue is not repro in high contrast aquatic theme. 
Similar issue is observed in 'View in OneNote button'.
Expected Result:
In high contrast desert theme, keyboard focus should be visible on the sign in options.
User Impact:
High contrast users unaware about the exact keyboard focus.
MAS Reference:
[MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)
